lib/app.js

```diff
@@ -14,38 +12,29 @@ import {
   AppMenu
 } from 'otp-react-redux'
 
+import {loadCustomIcons} from './util'
+
 // Loads a yaml config file which is set in the webpack.config.js file. This
 // setting is defined from a custom environment setting passed into webpack or
 // defaults to ./config.yml
 const otpConfig = require(YAML_CONFIG)
 
+// Loads those custom icons (in './icons' dir) specified by otpConfig.
+const customIcons = loadCustomIcons(otpConfig)
```

Comment:
I think we should get rid of the custom icons entirely. The way forward for defining your own custom set of icons is to pass in a custom LegIcon and ModeIcon component.

Reply:
@evansiroky, we need to retain custom icons because customization file `config.js` is not able to import any icons, according to [this comment](https://github.com/ibi-group/trimet-mod-otp/blob/master/lib/config.js#L9).

Reply:
Hmm... perhaps you're right, but we should at least have a place to put custom LegIcon/ModeIcons in this repo, don't you think?

Reply:
It is possible to inline a custom LegIcon/ModeIcon class by importing stuff from the otp-ui package or other packages. We do not need customIcons.

Reply:
The old customIcons have been entirely removed now.